src/main/kotlin/gg/essential/universal/wrappers/message/UMessage.kt

```diff
@@ -121,3 +121,21 @@ class UMessage {
         messageParts.forEach { _chatMessage.appendSibling(it) }
     }
 }
+
+private val printChatMessageWithOptionalDeletion: MethodHandle? = try {
+    val method = GuiNewChat::class.java.declaredMethods.find { method ->
+        method.parameterTypes.run {
+            size == 2 &&
+                    get(0) == MCITextComponent::class.java &&
+                    get(1) == Int::class.java
+        }
+    } ?: throw NoSuchMethodException("Couldn't find the right method for UMessage")
```

Comment:
The error should mention which method it was searching for, where it was searching, and why it needs it. Imagine reading this in a random log a year from now with absolutely no recollection of ever writing it.
E.g. `"Could not find method to edit chat messages. No method with parameters (${MCITextComponent::class.java.name}, int) in ${GuiNewChat::class.java}."`